src/main/java/mate/academy/spring/service/UserService.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    Optional<User> findById(Long id);
```

Comment:
from dto you return Optional, from service return User
```suggestion
    User findById(Long id);
```